service/app/distribution.go

```diff
@@ -222,9 +246,14 @@ func (dist Distribution) SlotCount() int {
 
 // SetCommitmentHash should
 // - validate the pack
+// - seal the pack
```

Comment:
add `TODO` (?)

Reply:
Actually sealing is done elsewhere (when we get the FlowID from the chain) so this should be removed. Thanks!